### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by michelson**

### CODE REVIEW

File: lib/active_storage.ex [hunk 1]
```diff
@@ -0,0 +1,257 @@
+defmodule ActiveStorage do
```
Comment (michelson) [inline lib/active_storage.ex:1]: this is the main context aka Phoenix Ecto context

File: config/config.exs [hunk 2]
```diff
@@ -21,6 +21,45 @@ use Mix.Config
...
+
+# Configure identify command:
+
+config :mogrify,
+  identify_command: [
+    path: "magick",
+    args: ["identify"]
+  ]
+
+
```
Comment (michelson) [inline config/config.exs:62]: app configuration

File: config/config.exs [hunk 3]
```diff
@@ -21,6 +21,45 @@ use Mix.Config
 #     config :logger, level: :info
 #
 
+# Use rails credentials:edit to set the AWS secrets (as aws:access_key_id|secret_access_key)
+config :chaskiq, :storage,
```
Comment (cheerfulstoic) [inline config/config.exs:25]:
```suggestion
config :active_storage_ex, :storage,
```

File: lib/active_storage.ex [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+              content_type: _content_type,
+              filename: _filename
+            } = input
+        },
+        %{context: %{current_user: _current_user}}
+      ) do
+    direct_upload_response = %{direct_upload: create_direct_upload(input)}
+
+    {:ok, direct_upload_response}
+  end
```
Comment (cheerfulstoic) [inline lib/active_storage.ex:31]: Maybe this should be pulled out into `chaskiq`?

**Review by cheerfulstoic — APPROVED** [review]
👍  Just a couple of comments, but I think you're still in the process of separating things out, which is fine.

Going to merge this so that I can work off `main`

File: lib/active_storage.ex (hunk 4 above)
Comment (cheerfulstoic) [reply lib/active_storage.ex:31]: Ah, I get it now.  It's an integration with absinthe.  We might rename it to `absinthe_resolver`.  Or we could just put the code for the resolver in the `README`.  That way apps could use it / modify it themselves.

Comment (michelson) [reply lib/active_storage.ex:31]: yeah, this could be just documented on "Absinthe integration" 👍

**Pull request merged**